org.gnome.Gtranslator.yml

```diff
@@ -21,14 +15,11 @@ cleanup:
   - '*.a'
 finish-args:
   - '--share=ipc'
-  - '--socket=x11'
+  - '--socket=fallback-x11'
```

Comment:
You snuck in a lot of unrelated changes into a single commit which I'm not a fan of.

Reply:
I agree that I should split this a bit